src/main/java/com/vmtapp/enterprise/VarmentTmsApplication.java

```diff
@@ -2,8 +2,11 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cache.annotation.EnableCaching;
 
 @SpringBootApplication
```

Comment:
add @EnableCaching annotation to make the application a better performance because the user may ask for ticket information multiple times. Source:https://martinfowler.com/articles/microservices.html#EvolutionaryDesign